pkg/controllers/gacha-controller.go

```diff
@@ -141,59 +143,46 @@ func HandleGachaDraw(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
 		return
 	}
-	characterPool := generateCharacterPool(characters)
+	characterPool := GenerateCharacterPool(characters)
```

Comment:
What is the reason for capitalization?

Reply:
There is no reason, I just liked it that way :smile:

Reply:
@iamananya 
In Golang, case has meaning.
You need to look up the Golang specs.

Reply:
@GuiltyMorishita Since this function is not exported, I will make it lowercase.